_posts/2015-09-27-october-meetup.md

```diff
@@ -0,0 +1,21 @@
+---
+title: CopenhagenJS October Meetup
+layout: post
+author: Kevin Simper
+date: Thursday, 2015-10-15 18:00
+lanyrd: http://lanyrd.com/2015/copenhagenjs-october/
+host: One.com
+host_url: https://www.one.com/
+host_address: Kalvebod Brygge 24, 1650 København
+---
+
+## Welcome to the October 2015 Meetup
+
+This meetup will be held at the One.com office, which is just south of the
+central station. This time we have 3 really promising javascript talks.
+
+The creater behind browser package manager called [jspm](http://jspm.io/) and
```

Comment:
Minor language corrections: 

The creator of the [jspm](http://jspm.io/) package manager and [systemjs](https://github.com/systemjs/systemjs) is visting Copenhagen and will give a talk.